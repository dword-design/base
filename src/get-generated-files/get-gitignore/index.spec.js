import { Base } from '@/src/index.js';

export default {
  valid() {
    expect(
      new Base({ gitignore: ['foo'] }).getGitignoreConfig(),
    ).toMatchSnapshot(this);
  },
};
