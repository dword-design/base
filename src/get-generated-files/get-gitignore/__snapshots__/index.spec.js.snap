// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index valid 1`] = `
Array [
  ".DS_Store",
  "/.env.json",
  "/.eslintignore",
  "/.nyc_output",
  "/.test.env.json",
  "/coverage",
  "/node_modules",
  "foo",
]
`;
